geode-core/src/integrationTest/java/org/apache/geode/internal/AvailablePortHelperIntegrationTest.java

```diff
@@ -266,6 +268,10 @@ public void getRandomAvailableTCPPort_returnsUniqueValues(final boolean useMembe
   @Parameters({"true", "false"})
   public void initializeUniquePortRange_willReturnSamePortsForSameRange(
       final boolean useMembershipPortRange) {
+    assumeTrue(
+        "Windows has ports scattered throughout the range that makes this test difficult to pass consistently",
+        SystemUtils.isWindows());
```

Comment:
seems like you intended to skip this test on windows, shouldn't it be `assumeFalse` instead?

Reply:
Doh! Thanks.